chart/templates/webserver/webserver-deployment.yaml

```
@@ -160,6 +163,7 @@ spec:
         - name: webserver
           image: {{ template "airflow_image" . }}
           imagePullPolicy: {{ .Values.images.airflow.pullPolicy }}
+          securityContext: {{ $containerSecurityContext or .Values.webserver.securityContexts.container .Values.securityContexts.container | nindent 12 }}
```

```suggestion
          securityContext: {{ or $containerSecurityContext .Values.webserver.securityContexts.container .Values.securityContexts.container | nindent 12 }}
```

Getting the following error in my test setup without the proposed change:
```
Error: template: test-airflow/charts/airflow/templates/webserver/webserver-deployment.yaml:166:30: executing "test-airflow/charts/airflow/templates/webserver/webserver-deployment.yaml" at <$containerSecurityContext>: can't give argument to non-function $containerSecurityContext

```

`helm version` output:
```
version.BuildInfo{Version:"v3.11.0", GitCommit:"472c5736ab01133de504a826bd9ee12cbe4e7904", GitTreeState:"clean", GoVersion:"go1.18.10"}

```

This is probably due to what is explained [here](https://github.com/helm/helm/issues/7711#issuecomment-593113347)

---

@ChrisFraun Hey! any chance to look at the proposed changes? Thanks!

---

sure! Thanks for the suggest!